.github/workflows/linux.yml

```diff
@@ -467,16 +460,24 @@ jobs:
           path: ${{ env.BUILD_DIR }}/genai_nodejs_bindings.tar.gz
           if-no-files-found: 'error'
 
-      - name: Run npm package tests
+      - name: Install npm package tests dependencies
         working-directory: ${{ env.SRC_DIR }}/src/js
         run: |
-          cp -R ${{ env.OV_INSTALL_DIR }}/nodejs bin
-          npm install
-          npm test
+          cp -R ${{ env.INSTALL_DIR }} bin
+          npm install --verbose
+          rm -rf node_modules/openvino-node/bin
+          cp -R ${{ env.OV_INSTALL_DIR }}/openvino_js_package node_modules/openvino-node/bin
```

Comment:
That's the first mention of `node_modules` folder in the file. Can you add an explanation to the yaml comment what it is for? This step isn't described in the instruction

Reply:
I added comments here.

Reply:
Oh, I see. Sure, let's do that.